### PR TITLE
Fix conflict of testing requirements

### PR DESCRIPTION
**Pull request by mauricioabreu**
`pytest-cov 2.6.0` has increased the version requirement for the coverage package from `>=3.7.1` to `>=4.4`, which is in conflict with the version requirement defined by the python-coveralls package for `coverage==4.0.3`.

Once `coveralls` upgrades its `coverage` dependency we can get back to a latest pytest-cov version

### CODE REVIEW

File: .venv/.Python [hunk 1]
```diff
@@ -0,0 +1 @@
+/System/Library/Frameworks/Python.framework/Versions/2.7/Python
```
Comment (leandromoreira) [inline .venv/.Python]: Is this related only for macos?

Comment (mauricioabreu) [reply .venv/.Python]: doh! my bad 😞 
I added it unintentionally
